app/src/main/java/org/sopt/sample/SignUpActivity.kt

```diff
@@ -0,0 +1,36 @@
+package org.sopt.sample
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import com.google.android.material.snackbar.Snackbar
+import org.sopt.sample.databinding.ActivitySignUpBinding
+
+class SignUpActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySignUpBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignUpBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        binding.btnChecksignup.setOnClickListener{
+            if(binding.etId.text == null || binding.etPw.text == null || binding.etMbti.text == null){
+                Toast.makeText(this, "회원 정보가 입력되지 않았습니다.", Toast.LENGTH_SHORT).show()
+            }
```

Comment:
코틀린 함수인 nullOrBlank()을 사용하시면 좀 더 코틀린스럽게 ㅎㅎ 사용할 수 있습니다! 좀 더 직관적이고요!

```suggestion
        binding.btnChecksignup.setOnClickListener{
            if(!binding.etId.text.isNullOrBlank() && !binding.etPw.text.isNullOrBlank()&& !binding.etPw.text.isNullOrBlank())){
                Toast.makeText(this, "회원 정보가 입력되지 않았습니다.", Toast.LENGTH_SHORT).show()
            }
```